projects/batfish/src/test/java/org/batfish/grammar/cisco/CiscoGrammarTest.java

```diff
@@ -196,6 +198,77 @@ public void testBgpRemovePrivateAs() throws IOException {
     assertFalse(r3HasPrivate);
   }
 
+  @Test
+  public void testOspfPointToPoint() throws IOException {
+    String testrigName = "ospf-point-to-point";
+    String iosOspfPointToPoint = "ios-ospf-point-to-point";
+    String[] configurationNames = new String[] {iosOspfPointToPoint};
+    Batfish batfish =
+        BatfishTestUtils.getBatfishFromTestrigResource(
+            TESTRIGS_PREFIX + testrigName, configurationNames, null, null, null, null, _folder);
+    batfish.getSettings().setDisableUnrecognized(false);
+    SortedMap<String, Configuration> configurations;
+    try {
+      configurations = batfish.loadConfigurations();
+    } catch (CompositeBatfishException e) {
+      throw e.asSingleException();
+    }
+    Configuration iosMaxMetric = configurations.get(iosOspfPointToPoint);
+    Interface e0Sub0 = iosMaxMetric.getInterfaces().get("Ethernet0/0");
+    Interface e0Sub1 = iosMaxMetric.getInterfaces().get("Ethernet0/1");
+
+    assertTrue(e0Sub0.getOspfPointToPoint());
+    assertFalse(e0Sub1.getOspfPointToPoint());
+  }
+
+  @Test
+  public void testOspfMaxMetric() throws IOException {
+    String testrigName = "ospf-max-metric";
+    String iosMaxMetricName = "ios-max-metric";
+    String iosMaxMetricCustomName = "ios-max-metric-custom";
+    String iosMaxMetricOnStartupName = "ios-max-metric-on-startup";
+    String[] configurationNames =
+        new String[] {iosMaxMetricName, iosMaxMetricCustomName, iosMaxMetricOnStartupName};
+    Batfish batfish =
+        BatfishTestUtils.getBatfishFromTestrigResource(
+            TESTRIGS_PREFIX + testrigName, configurationNames, null, null, null, null, _folder);
+    batfish.getSettings().setDisableUnrecognized(false);
+    SortedMap<String, Configuration> configurations;
+    try {
+      configurations = batfish.loadConfigurations();
+    } catch (CompositeBatfishException e) {
+      throw e.asSingleException();
+    }
+    Configuration iosMaxMetric = configurations.get(iosMaxMetricName);
+    Configuration iosMaxMetricCustom = configurations.get(iosMaxMetricCustomName);
+    Configuration iosMaxMetricOnStartup = configurations.get(iosMaxMetricOnStartupName);
+    OspfProcess proc = iosMaxMetric.getDefaultVrf().getOspfProcess();
+    OspfProcess procCustom = iosMaxMetricCustom.getDefaultVrf().getOspfProcess();
+    OspfProcess procOnStartup = iosMaxMetricOnStartup.getDefaultVrf().getOspfProcess();
+    long expectedMaxMetricRouterLsa =
+        org.batfish.representation.cisco.OspfProcess.MAX_METRIC_ROUTER_LSA;
+    long expectedMaxMetricStub = org.batfish.representation.cisco.OspfProcess.MAX_METRIC_ROUTER_LSA;
+    long expectedMaxMetricExternal =
+        org.batfish.representation.cisco.OspfProcess.DEFAULT_MAX_METRIC_EXTERNAL_LSA;
+    long expectedMaxMetricSummary =
```

Comment:
I wouldn't typically expect you to introduce local constants that exactly match public constants, but ok.

Reply:
It was just to decrease verbosity. Could have used static imports but meh.